changedetectionio/tests/test_access_control.py

```diff
@@ -68,11 +65,8 @@ def test_check_access_control_no_blank_password(app, client):
         # Enable password check.
         res = c.post(
             url_for("settings_page"),
-            data={"password": "",
-                  "minutes_between_check": 180,
-                  'fetch_backend': "html_requests"},
-
-        follow_redirects=True
+            data={"password": "", "minutes_between_check": 180, "fetch_backend": "html_requests"},
```

Comment:
I prefer it how it was - line by line, makes it easier to see what the params are I think